assets/src/components/swingsView.tsx

```diff
@@ -22,11 +22,16 @@ const SwingsView = (): ReactElement<HTMLElement> => {
   const currentTime = useCurrentTime()
   const swings = useSwings()
 
-  const activeSwings = swings
-    ? swings.filter((swing) => {
-        return swing.time + 600 > serviceDaySeconds(currentTime)
-      })
-    : []
+  const [activeSwings, pastSwings] = swings
+    ? partition(
+        swings.sort((swing1, swing2) => {
+          return swing1.time - swing2.time
+        }),
+        (swing) => {
+          return swing.time + 600 > serviceDaySeconds(currentTime)
```

Comment:
@lemald let's make this 600 a constant, since I'm going to change it anyhow in another PR.